update-css/sw.js

```diff
@@ -0,0 +1,108 @@
+var cacheName = 'update-css-2015.1015.1202';
+
+self.addEventListener('install', function(ev) {
```

Comment:
We've been using `event` instead of `ev` everywhere else, may be best to be consistent.